app/main.py

```diff
@@ -1,2 +1,103 @@
+from __future__ import annotations
+from typing import Any, Iterator
+
+
+class Node:
+    def __init__(self, key: Any, key_hash: int, value: Any) -> None:
+        self.key = key
+        self.key_hash = key_hash
+        self.value = value
+
+
 class Dictionary:
-    pass
+    def __init__(self, capacity: int = 8) -> None:
+        self.capacity = capacity
+        self.load_factor = 2 / 3
+        self.threshold = self.capacity * self.load_factor
+        self.nodes = [None] * self.capacity
```

Comment:
Would make sense to call it `self.hash_table` + to add type annotation that it is a list of nodes